seed/views/properties.py

```diff
@@ -324,6 +329,33 @@ def _move_relationships(self, old_state, new_state):
 
         return new_state
 
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_viewer')
+    def list(self, request):
```

Comment:
it's a bummer that you had to add this back in.

Reply:
Technically we don't need it, and we're not actually using it, but I think it's helpful for documenting the use of passing `-1` as a profile id to return all columns